src/UserService.Models.Dto/Requests/User/Images/RemoveImagesRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LT.DigitalOffice.UserService.Models.Dto.Enums;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.UserService.Models.Dto.Requests.User.Images
+{
+  public record RemoveImagesRequest
+  {
+    public Guid EntityId { get; set; }
```

Comment:
Is it entity that have images with theese imagesIDs? why do we need this property?

Reply:
If it is user - to remove avatar from user table
Dont know, would it be useful for certificates and educations or not

Reply:
also now we remove images only for one entity per request